### DISCUSSION TITLE
Multiples registers

### DISCUSSION THREAD

**Discussion by Sephiroth80** [poweradmin/poweradmin#590] [General]
Hi i have a question
there is a posibility to add multiple registry on the gui?
actualy i can put only one, press add record, another, etc.
nothing more
thanks a lot

**Comment by edmondas** [comment]
@Sephiroth80 Not sure I understand what you mean by registry, is it domain or record? There's bulk domain function, but regarding DNS records those can only be added one at a time or use templates to have a predefined set of those.

**Reply by Sephiroth80** [reply]
thanks, but thats what i need
charge multiple records at the same time.
![Captura](https://github.com/user-attachments/assets/69deb145-ae52-4ef2-9152-97e3113a47c9)
for example, next to the boton add record, have another botton can say "add field" and apear a new field to put a new record.
Thanks you for answer me.

**Comment by edmondas** [comment]
Added cross-reference to #106 to track requirement

**Comment by edmondas** [comment]
Decided not to expand the zone edit form due to complexity. Added a "multi-record mode" instead, where rows can be added/removed - see implementation ticket #106.
